### ISSUE TITLE
importing Dat.app causes freeze

### ISSUE THREAD

**Issue by maxogden** [juliangruber/hyperdrive-import-files#48]
I downloaded the dat desktop mac app build from here https://github.com/datproject/dat-desktop/releases/tag/untagged-4024b3636b599fbc021c and then unzipped it and decided to use that folder as a test dat but ran into a problem where the import froze at 461/1181 files.

Importing just the zip worked ok so it must be some weird file in the Dat.app

```
~/Desktop 🐈  cd dat-desktop/
~/Desktop/dat-desktop 🐈  ls
Dat.app				dat-desktop-1.0.2-mac.zip
~/Desktop/dat-desktop 🐈  dat create
Dat Archive created: /Users/max/Desktop/dat-desktop
Link: dat://a629d98f574152348888c2fcb782c6b09385667f627ab384713b71be895159f6
~/Desktop/dat-desktop 🐈  dat sync
Syncing Dat Archive: /Users/max/Desktop/dat-desktop
Link: dat://a629d98f574152348888c2fcb782c6b09385667f627ab384713b71be895159f6

Importing files...
[===========================================>-------] 461 of 1181 files (1.73 MB/s)

Looking for connections in Dat Network...


```

The way I know it froze was the speed was slowly dropping over time, from 5MB/s down to 1.73 when I took the screenshot, and no visible progress was being made

**Comment by juliangruber** [comment]
This doesn't seem to be a problem with this module itself, or at least not on my machine. You can verify hyperdrive-import-files only logic running this:

```bash
$ node example.js /Applications/Dat.app/
```

**Comment by juliangruber** [comment]
I also tested with leveldb as a backend, instead of memdb, and it also passes.

**Comment by juliangruber** [comment]
I can verify however that with `dat(1)` it halts after importing `1146 of 1180 files`

**Comment by joehand** [comment]
I can look into this. Mine got stuck too on file 461, the same as Max. 

I took the problem file out and it still didn't seem to finish (but it didn't get stuck). It also looks like the progress is off. If I run debug on this it shows the total size as 350MB but actual size is 3.2G.

Not sure if it matters, but the file I got stuck on was:

```
Contents/Resources/app.asar.unpacked/node_modules/fsevents/build/action_after_build.target.mk
```

Since this seems to import okay, moved this to dat-node https://github.com/datproject/dat-node/issues/103.
